helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -538,7 +538,12 @@ public void invoke(NotificationContext changeContext) throws Exception {
     }
   }
 
-  private void subscribeChildChange(String path, NotificationContext.Type callbackType) {
+  /*
+   * If callback type is INIT or CALLBACK, subscribes child change listener to the path
+   * and returns the path's children names. The children list might be null when the path
+   * doesn't exist or callback type is other than INIT/CALLBACK/FINALIZE.
```

Comment:
The current logic will return a list even the type is not INIT/CALLBACK/FINALIZE, right? Although I guess it is expected.
1. Please fix the comment.
2. The method name becomes misleading somehow with this change. I guess it should be called getChildrenAndSubscribeChildChange. Obviously, not a good name. But feel free to change if you have a better idea. This is just a nice to have thing.

Reply:
Resolved comment.
Since we also have `subscribeChildChanges()` in ZkClient that returns children, so I'd like to keep it as it is to be consistent. Besides, I don't have a better name for this and I think it is OK to keep it.